File: EVA2_PRACTICA_07.java

import java.util.Scanner;

/*
 * Escribir un programa que lea la temperatura en grados centigrados, y muestre
 * un mensaje acorde a la temperatura
 */

/**
 *
 * @author riguidix
 */
public class Principal {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        // TODO code application logic here
        int iTemperatura;
        Scanner input = new Scanner(System.in);
        
        System.out.println("Introduzca la temperatura");
        iTemperatura = input.nextInt();
        
        if (iTemperatura <= 0) {
            System.out.println("Freezing Weather");
        } else if (iTemperatura <= 10) {
            System.out.println("Very Cold Weather");
        } else if (iTemperatura <= 20) {
            System.out.println("Cold Weather");
        } else if (iTemperatura <= 30) {
            System.out.println("Normal in Temp");
        } else if (iTemperatura <= 40) {
            System.out.println("It's Hot");
        } else if (iTemperatura >= 40) {
            System.out.println("It's very Hot");
        }
    }
    
}

File: EVA2_PRACTICA_07.py
iTemperatura = int(input('Introduzca la temperatura: '))

if (iTemperatura <= 0) :
    print('Freezing Weather')
elif (iTemperatura <= 10) :
    print('Very Cold Weather')
elif (iTemperatura <= 20) :
    print('Cold Weather')
elif (iTemperatura <= 30) :
    print('Normal in Temp')
elif (iTemperatura <= 40) :
    print("It's Hot")
elif (iTemperatura >= 40) :
    print("It's Very Hot")